recipes-test/demo-network-config/files/26-multihomed-client.network

```diff
@@ -0,0 +1,9 @@
+[Match]
+Name=@IFNAME@
+
+[Network]
+Description=Multihomed network. DHCP-assigned IP for Primary<->Backend. Statically assigned IP for Primary<->Secondary
```

Comment:
How is multihoming actually being used here? I'm not sure I'm following it.

Reply:
Two IP addresses are assigned two a single RPi Ethernet NIC. One is statically defined (10.0.3.1) and  aimed for communication with secondaries, the second is obtained from DHCP server running on a local router that has an access to Inet thus it allows Primary to communicate with the server.

Reply:
Thanks, I'm starting to see it more clearly. But in the RPi case, if you only have one ethernet connection and you aren't using wifi, how do you access multiple networks? Are you plugging the primary and secondary RPis both into the same router or switch so that they can talk to each other through that without leaving the local network?

Reply:
Yes, I plug them into the same switch that works just on data link layer (layer 2) so it doesn't impact IP level anyhow which allows my to configure as many as I want IP networks  for devices connected to it. I'll prepare some doc depicting networking topology for this setup later on.